frontend/src/pages/dashboard/betas/create/index.tsx

```diff
@@ -2,12 +2,14 @@ import { Box } from '@mui/material';
 import { Stack } from '@mui/system';
 import { useNavigate } from 'react-router';
 import { BetaCreateEditFormValues } from 'src/@types/beta';
-import BackBar from '../../../../components/BackBar';
-import { PATH_DASHBOARD } from '../../../../routes/paths';
-import { postCreateBeta, uploadBetaVideoToCloudfare } from '../../../../services/betas';
-import { useDispatch } from '../../../../store';
-import { openMessageBar } from '../../../../store/reducers/messageBar';
-import { refreshView } from '../../../../store/reducers/ui';
+import BackBar from 'src/components/BackBar';
+import { PATH_DASHBOARD } from 'src/routes/paths';
+import { postCreateBeta, uploadBetaVideoToCloudfare } from 'src/services/betas';
```

Comment:
Nit: function names are breaking the abstraction barrier. Clients do not need to know the implementation details. Introducing more assumptions to our service layer such as using POST to create and we are uploading to Cloudfare will lead to extensibility issues in the future and may potentially break the code.

Reply:
proposal: 
postCreateBeta -> createBetaVideo
uploadBetaVideoToCloudfare -> uploadBetaVideo

Reply:
agree to be fixed in next pr

Reply:
For the part on uploadBetaVideo I chose to include the service provider name, in case we are changing our video service provider in the future.

Reply:
Agree with createBeta